parlai/agents/image_seq2seq/image_seq2seq.py

```diff
@@ -55,6 +55,7 @@ def add_cmdline_args(cls, argparser):
             recommended=True,
             help='if true, include image token (or no image token) for each example',
         )
+        return group
```

Comment:
does this change belong here?

Reply:
yes, my repetitionunlikelihood agent inherits from image_seq2seq because we used an image_seq2seq model as our ELI5 baseline. when I call super().add_cmdline_args() from there, I get None without this line